### DISCUSSION TITLE
Messages count ?

### DISCUSSION THREAD

**Discussion by Abacaxi-Nelson** [GetStream/stream-chat-flutter#594] [Q&A] [answered]
Hi all,
Is there a way to get the number of message of the channel ? even more, a stream ?

im trying to reproduce YouTube chat, and want to display it

Thanks,
Nelson

**Comment by imtoori** [comment, marked as answer]
Hey Nelson :)
Unfortunately, that's not possible at the moment. 
It's a backend limitation
